lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2889,12 +2889,14 @@ the column index.
           <xs:annotation>
             <xs:documentation xml:lang="en">This attribute is used only if
 ``type`` attribute value is ``select`` - render a select list as a set of check
```

Comment:
I am going to open a separate PR to standardise a bit these attribute descriptions, adding my changes for these ones here so we don't have conflicts.

```suggestion
            <xs:documentation xml:lang="en">Render a select list as a set of
checkboxes (``checkboxes``) or radio buttons (``radio``; note this is
```

Reply:
The mentioned PR is https://github.com/galaxyproject/galaxy/pull/11390 , if you want to take a look!